From: Modestas Vainius <modestas@vainius.eu>
Subject: [PATCH] Look for libmysqld_pic

Debian package libmysqld-pic ships libmysqld -fPIC version as
libmysqld_pic. Hence look for it too in FindMySQLAmarok.cmake.

Signed-off-by: Modestas Vainius <modestas@vainius.eu>

---
 cmake/modules/FindMySQLAmarok.cmake |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/cmake/modules/FindMySQLAmarok.cmake b/cmake/modules/FindMySQLAmarok.cmake
index be0a478..cdefc0d 100644
--- a/cmake/modules/FindMySQLAmarok.cmake
+++ b/cmake/modules/FindMySQLAmarok.cmake
@@ -40,7 +40,7 @@ find_library(MYSQL_LIBRARIES NAMES mysqlclient
    /opt/ports/lib/mysql5/mysql
 )
 
-find_library(MYSQL_EMBEDDED_LIBRARIES NAMES mysqld libmysqld
+find_library(MYSQL_EMBEDDED_LIBRARIES NAMES mysqld_pic mysqld libmysqld
    PATHS
    ~/usr/lib/mysql
    /opt/local/lib/mysql5/mysql
-- 
tg: (d5becae..) debian/mysqle_look_for_libmysqld_pic (depends on: upstream)
